modin/core/storage_formats/cudf/parser.py

```diff
@@ -126,7 +123,10 @@ def parse(cls, fname, **kwargs):
             index = len(pandas_df)
         partition_dfs = _split_result_for_readers(1, num_splits, pandas_df)
         key = [
-            put_func(GPU_MANAGERS[gpu_selected], partition_df)
+            put_func(
+                cls.frame_partition_mgr_cls._get_gpu_managers()[gpu_selected],
+                partition_df,
+            )
```

Comment:
Avoids circular imports.
In general, it's best not to rely on implementation details (`GPU_MANAGERS`).

Reply:
not actual